src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp

```diff
@@ -188,15 +188,15 @@ class OpenGLDrawingEngine final : public IDrawingEngine
     uint32_t _height = 0;
     uint32_t _pitch = 0;
     size_t _bitsSize = 0;
-    uint8_t* _bits = nullptr;
+    std::unique_ptr<uint8_t> _bits;
```

Comment:
I'm not convinced this works as expected. You replaced a pointer to an arbitrary number of `uint8_t` with a pointer to a single value. You probably intended this to be `std::unique_ptr<uint8_t[]>`.

Reply:
In general I would replace it with a std::vector instead of a unique ptr.

Reply:
IMO use of `std::vector` implies resizability, if this array stays at a constant size then it shouldn't be needed.

Reply:
I think it only ever gets resized when the window size changes, either way works, the benefit of vector is that you will probably not re-allocate when you change the window size often but that doesn't matter as much here.